numba_cuda/numba/cuda/tests/cudadrv/test_host_alloc.py

```diff
@@ -4,10 +4,10 @@
 import numpy as np
 from numba.cuda.cudadrv import driver
 from numba import cuda
-from numba.cuda.testing import unittest, ContextResettingTestCase
+from numba.cuda.testing import unittest, CUDATestCase
 
 
-class TestHostAlloc(ContextResettingTestCase):
+class TestHostAlloc(CUDATestCase):
```

Comment:
A lot of these are fairly old tests from the upstream numba code base where it looks like a lot of tests may have simply reset the device every time in an effort to be extra safe and start with a clean slate every time. This may have made sense at the time but might not anymore.